recipes/IEMOCAP/emotion_recognition/hparams/train_with_wav2vec2.yaml

```diff
@@ -0,0 +1,116 @@
+# ########################################
+# Emotion recognition from speech using wav2vec2
+#
+#  * Authors: Yingzhi WANG
+# ########################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1968
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+experiment: emotion_recognition
+# Set up folders for reading from and writing to
+# Dataset will be downloaded to the `data_folder`
+data_folder: !PLACEHOLDER # e.g., /storage/IEMOCAP/IEMOCAP_ahsn_leave-two-speaker-out
+output_folder: !ref results/wav2vec/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# URL for the wav2vec2 model, you can change to benchmark diffrenet models
+wav2vec2_hub: "facebook/wav2vec2-base-960h"
+
+# different speakers for train, valid and test sets
+# (more challenging exercise: higher error rate)
+different_speakers: False
+
+# Path where data manifest files will be stored
+train_annotation: !ref <output_folder>/train.json
+valid_annotation: !ref <output_folder>/valid.json
+test_annotation: !ref <output_folder>/test.json
+
+# The train logger writes training statistics to a file, as well as stdout.
+train_logger: !new:speechbrain.utils.train_logger.FileTrainLogger
+    save_file: !ref <train_log>
+
+ckpt_interval_minutes: 15 # save checkpoint every N min
+
+# Training parameters
+number_of_epochs: 30
+batch_size: 4
+lr: 0.0001
+lr_wav2vec2: 0.00001
+
+#freeze all wav2vec2
+freeze_wav2vec2: False
+#set to true to freeze the CONV part of the wav2vec2 model
+freeze_wav2vec2_conv: True
```

Comment:
@BenoitWang , do you try unfreezing conv layer of wav2vec ?
you got bad number ?

Reply:
yes unfreeze conv shows a little bit worse but I am carring out more tests to be precise